.circleci/config.yml

```diff
@@ -0,0 +1,133 @@
+  # If any of these refer to each other, rewrite them so that they don't or see https://circleci.com/docs/2.0/env-vars/#interpolating-environment-variables-to-set-other-environment-variables .
+  environment:
+    CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+    CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+  # In CircleCI 1.0 we used a pre-configured image with a large number of languages and other packages.
+  # In CircleCI 2.0 you can now specify your own image, or use one of our pre-configured images.
+  # The following configuration line tells CircleCI to use the specified docker image as the runtime environment for you job.
+  # We have selected a pre-built image that mirrors the build environment we use on
+  # the 1.0 platform, but we recommend you choose an image more tailored to the needs
+  # of each job. For more information on choosing an image (or alternatively using a
+  # VM instead of a container) see https://circleci.com/docs/2.0/executor-types/
+  # To see the list of pre-built images that CircleCI provides for most common languages see
+  # https://circleci.com/docs/2.0/circleci-images/
+  docker:
+  - image: circleci/build-image:ubuntu-14.04-XXL-upstart-1189-5614f37
+    command: /sbin/init
+
+jobs:
+  build:
+    <<: *defaults
+    steps:
+    # Machine Setup
+    #   If you break your build into multiple jobs with workflows, you will probably want to do the parts of this that are relevant in each
+    # The following `checkout` command checks out your code to your working directory. In 1.0 we did this implicitly. In 2.0 you can choose where in the course of a job your code should be checked out.
+    - checkout
+    # Prepare for artifact and test results  collection equivalent to how it was done on 1.0.
+    # In many cases you can simplify this from what is generated here.
+    # 'See docs on artifact collection here https://circleci.com/docs/2.0/artifacts/'
+    - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+    # This is based on your 1.0 configuration file or project settings
+    - run:
+        working_directory: ~/all-of-us/raw-data-repository
+        command: 'sudo service mysql status || sudo service mysql start; '
+    # Dependencies
+    #   This would typically go in either a build or a build-and-test job when using workflows
+    # Restore the dependency cache
+    - restore_cache:
+        keys:
+        # This branch if available
+        - v1-dep-{{ .Branch }}-
+        # Default branch if not
+        - v1-dep-master-
+        # Any branch if there are none on the default branch - this should be unnecessary if you have your default branch configured correctly
+        - v1-dep-
+    # This is based on your 1.0 configuration file or project settings
+    - run: sudo chmod o+w /opt
+    - run: pip install --upgrade pip
+    - run: pip install --upgrade httplib2 jira oauth2client parallel pylint
+    - run: pip install -I alembic==0.9.1
+    # Save dependency cache
```

Comment:
Please remove commented out code if you don't think we need it.

Reply:
done